Lib/idlelib/pyshell.py

```diff
@@ -574,9 +574,9 @@ def transfer_path(self, with_cwd=False):
 
         self.runcommand("""if 1:
         import sys as _sys
-        _sys.path = %r
+        _sys.path = {!r}
         del _sys
-        \n""" % (path,))
+        \n""".format(path))
```

Comment:
Three strings could be multiline f strings, but .format good enough for now.

Reply:
Just a note: pyupgrade is intentionally cautious and doesn't upgrade multiline formats.

Reply:
I figured that.  I can change if I want next time I edit this file.

 A bit ironically, inserting .format on the first line of a 3 line call as it did twice in redirector.py messed up the PEP8 alignment of the arguments.  You are welcome to check my conversion to f-strings in that file.

Does pyupgrade pay any attention to line lengths?  I believe my patched lines are 80 chars.

Reply:
> Does pyupgrade pay any attention to line lengths? I believe my patched lines are 80 chars.

No, it only cares about semantics, not style. Generally people use `black` on top of `pyupgrade` to fix the style issues :)

Reply:
I prefer my styling ;-).

Reply:
Yeah, me too 👍 
Not a big `black` user.